testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -557,6 +557,7 @@
 ITP = resource_filename(__name__, 'data/gromacs_ala10.itp')
 ITP_nomass = resource_filename(__name__, 'data/itp_nomass.itp')
 ITP_atomtypes = resource_filename(__name__, 'data/atomtypes.itp')
```

Comment:
Ok the final test failing is that this needs to be added to ‘__all__’ (for reasons I’m unsure of)